src/index.html

```diff
@@ -17,35 +17,35 @@
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
+      <a class="header__logo" href="/">
```

Comment:
```suggestion
      <a class="header__logo" href="#">
```
change anywhere (check your demo link why)